src/coreclr/inc/corinfo.h

```diff
@@ -2112,6 +2112,21 @@ class ICorStaticInfo
         CORINFO_METHOD_HANDLE meth1Hnd,
         CORINFO_METHOD_HANDLE meth2Hnd) = 0;
 
+    //------------------------------------------------------------------------------
+    // getTypeDefinition: Get the (unconstructed) type definition from a given type handle.
+    //
+    // Arguments:
+    //    type - The input type handle
+    //
+    // Return Value:
+    //   The type handle for the (unconstructed) type definition from type.
+    //
+    // Remarks:
+    //   This is equivalent of Type.GetGenericTypeDefinition(). Given a generic type handle, it will
```

Comment:
This comment should be updated to match the curent implementation

Reply:
Fixed in https://github.com/dotnet/runtime/pull/103528/commits/6a86b7ce7e4cf63577f8d14c6ea69cc3d39bf9d7 👍